src/libs/algolia/fetchAlgolia/fetchOffersModules.ts

```diff
@@ -16,7 +16,7 @@ export const fetchOffersModules = async ({
   paramsList,
   userLocation,
   isUserUnderage,
-}: FetchMultipleOffersArgs): Promise<SearchResponse<Offer>[]> => {
+}: FetchMultipleOffersArgs): Promise<(SearchForFacetValuesResponse | SearchResponse<Offer>)[]> => {
```

Comment:
Je comprends que tu as juste "fait marcher" mais ça marcherait pas juste de supprimer ce typage ?
```suggestion
}: FetchMultipleOffersArgs) => {
```
TypeScript n'arriverait pas à s'en sortir ?

Reply:
Si je retire on a ça `Promise<(SearchForFacetValuesResponse | SearchResponse<unknow>)[]>`, c'est pour ça que c'est resté

Reply:
Vu ensemble, il faut juste passer `<Offer>` à `multipleQueries<Offer>(queries)` et tout va bien.